### ISSUE TITLE
Popup throws fatal error on window resize

### ISSUE THREAD

**Issue by codefactor** [SAP/ui5-webcomponents#2935]
**Describe the bug**
I am using the Dialog component and for some reason the Popup crashes with a fatal error any time I resize the screen.

**To reproduce**
1. Using the xweb/sample-widget repository, clone it and check out the `feature/WEF-10553` branch
2. Run `npm install`, then `npm start`, if you get a build error run `npm run build:tempfix`
3. Sample widget should open in browser and just resize the screen
4. Fatal error occurs

**Expected behavior**
The error should not happen

**Screenshots**
<img width="1290" alt="Screen Shot 2021-03-10 at 5 45 40 PM" src="https://user-images.githubusercontent.com/730310/110723179-80d7ca00-81c8-11eb-9dfd-99c99605dc80.png">

**Context**
- UI5 Web Components version : ^0.29.4
- OS/Platform: Mac

**Affected components** Popup.js Dialog.js

**Log output / Any errors** in the console
```
Popup.js:256 Uncaught TypeError: Cannot read property 'focus' of undefined
    at HTMLElement._onfocusout (Popup.js:256)
    at EventPart.handleEvent (parts.ts:532)
    at HTMLElement.EventPart.__boundHandleEvent (parts.ts:490)
```

**Organization:** SuccessFactors

**Priority:** *(optional)* Medium

Setting priority would indicate the importance of the issue
and would help the process of dispatching and processing.

To set the appropriate priority, you need to consider the impact
and if the issue:
- Breaks entire application or system - High and Very High
- Is accessibility one (Screan Reader, Keyboard Handling) - Medium and High
- Is functional one and makes a single component unusable - Medium and High
- Is visual, styling one - Low and Medium

Note: the priority would be re-evaluated by the issue processors, according to the project internal criterias

**Business impact:** *(mandatory for "Very High")*

**Comment by codefactor** [comment]
I think maybe this is only happening in 0.29.4 -- I tried delete package-lock.json and npm install, and the issue no longer happens. It could just be some other weird state on my local.

**Comment by codefactor** [comment]
Ok - now I'm getting the issue again - I feel it might have something to do with the `npm run build:tempfix` that was used to work around a different problem. That tempfix was the proposal to delete some src files from the npm script to get the build to run.

**Comment by codefactor** [comment]
I tried upgrading to 0.29.6 now and it seems to be better now - I'll update again if I run into it again

**Comment by ilhan007** [comment]
Ok, closing for now. Leave a comment and we will reopen the issue. Hubery Shen has also reported this issu, when he used older version, but he confirmed it's fixed in 0.29.6.
